INSTALL.md

```diff
@@ -222,7 +221,9 @@ be that you don't have the required C libraries installed.  See
 
 #### Install cabal-install
 
-The easiest way to get [cabal-install] is probably via your system's package
+Probably one of the best ways to get [cabal-install] is to use [GHCup], which is the main Haskell installer according to language's official [website][GHCup] and community [survey]. GHCup is available for all Linux distributions and is considered less error prone than other installation options.
```

Comment:
We might want to note that GHCup also works on the BSDs, which is relevant for XMonad

Reply:
The GHCup [website](https://www.haskell.org/ghcup/install/#supported-platforms) says that only FreeBSD officially supported so far. Work on other major BSDs is in [progress](https://github.com/haskell/ghcup-hs/issues/707) as I can see. Should I specify BSD-based operating systems in general or only FreeBSD?

Reply:
Maybe just add a link to the GHCup supported platforms page, like [this](https://github.com/xmonad/xmonad/pull/441/commits/2adeea6e2883d35d2002dc3dabc633c5221ed675)?

Reply:
How about "ghcup is widely available" with "widely available" being a link to that?

Reply:
Great, thanks! [Updated](https://github.com/xmonad/xmonad/pull/441/commits/064d5fac5222df449dc26277a0247ab27e6742ca).